app/assets/stylesheets/image_x.scss

```diff
@@ -71,3 +77,21 @@ $image-thumb-slider-border-color: #000;
 .#{$namespace}-image-x-label {
   text-align: center;
 }
+
+.frame {
+    position:absolute;
+    display:block;
+    border: 1px solid black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented 2 spaces, but was indented 4 spaces